frontend/gpay_agent/lib/business_verification_data/globals.dart

```diff
@@ -0,0 +1,15 @@
+library gpay_agent.globals;
+import 'package:agent_app/business_verification_data/verification.dart';
+
+final String merchantName = 'Medidoddi Vahini';
+final String address = 'H-no:15-108/1/k, Raghavendra colony, Nagarkurnool, Telangana';
+final String phone = '9381143249';
+
+enum verificationStatus{
```

Comment:
Why is this enum defined here?

Reply:
I want to access in both verification.dart and business_verification_home_view.dart to send data to back-end. So declared in globals.dart

Reply:
ok